WeScan/Extensions/AVCaptureSession+Utils.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  AVCaptureSession+Utils.swift
+//  WeScan
+//
+//  Created by David Steppenbeck on 2020/04/22.
+//  Copyright © 2020 WeTransfer. All rights reserved.
+//
+
+import AVFoundation
+
+extension AVCaptureSession {
+
+    /// Safely assigns the `sessionPreset` property.
+    ///
+    /// The assignment will only be made if the `canSetSessionPreset(_:)` method returns `true`.
+    ///
+    /// - Parameters:
+    ///   - preset: The `AVCaptureSession.Preset` value to be assigned.
+    func safeSetSessionPreset(_ preset: AVCaptureSession.Preset) {
```

Comment:
I don't think we're going to reuse this code in other places so I rather see this check being used directly in code

Reply:
Gotcha... just done.